lib/wash/exe/import.js

```diff
@@ -106,6 +121,14 @@ ImportCommand.prototype.import = function(destination) {
   document.body.appendChild(input);
 
   input.click();
+
+  // Localize the handler so we can remove it later.
```

Comment:
The handler removal code wasn't usable (because the event would often get triggered before we could cancel), but I thought the sentiment was still a good one (so that a removeEventListener could be used at some point) and it didn't create any additional code, so I kept it and the comment to explain why I was doing it like that.